WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -7,6 +7,9 @@ struct InboxNoteRow: View {
 
     // Tracks the scale of the view due to accessibility changes.
     @ScaledMetric private var scale: CGFloat = 1
+    @State private var displayedURL: URL?
+    @State private var showWebView: Bool = false
+    @State private var dismissButtonIsLoading: Bool = false
```

Comment:
super nit: `is*` might be more clear, but it's really minor and up to you 😄 
```suggestion
    @State private var isDismissButtonLoading: Bool = false
```